prov/verbs/src/verbs_info.c

```diff
@@ -361,7 +362,12 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 		return -FI_ENODATA;
 	}
 
-	if (attr->total_buffered_recv > info->rx_attr->total_buffered_recv) {
+	rm_enabled =(info->domain_attr &&
+		     info->domain_attr->resource_mgmt == FI_RM_ENABLED);
+
+	if (!rm_enabled &&
+	    (attr->total_buffered_recv > info->rx_attr->total_buffered_recv))
+	{
```

Comment:
hmm... let me do another iteration with understanding of the description :)

> When the domain supports FI_RM_ENABLED, the actual amount of buffering provided may exceed the value specified in total_buffered_recv.

So, this check has no meaning if FI_RM_ENABLED.

ofcause, I can set total_buffered_recv to -1 and the check may stay without modification, but I think updated patch does exactly what's described in the doc and matches to  real verbs/rdm behaviour, doesn't it?